Docker/build/WDQS-frontend/Dockerfile

```diff
@@ -36,7 +36,6 @@ COPY custom-config.json /templates/custom-config.json
 COPY default.conf /templates/default.conf
 
 ENV LANGUAGE=en\
-    BRAND_TITLE=DockerWikibaseQueryService\
```

Comment:
Shoudn't this just be renamed?

Reply:
Maybe. I had an issue overwriting this ENV while, when I set up the configuration, and so I removed it to avoid possible confusion, with having the default "DockerWikibaseQueryService" in two different places, here and in the .env file.